kubernetes-model-generator/generateModel.sh

```diff
@@ -74,6 +88,11 @@ echo "Installing required common modules"
 mvn clean install -f ../pom.xml -N
 mvn clean install -N
 mvn clean install -pl kubernetes-model-common -pl kubernetes-model-jsonschema2pojo
+mvn clean install -DskipTests 
+mvn clean install -f ../model-annotator/pom.xml
+mvn clean install -DskipTests -f ../kubernetes-client/pom.xml
+mvn clean install -DskipTests -f ../kubernetes-server-mock/pom.xml
+mvn clean install -DskipTests -f ../extensions/
```

Comment:
Only the annotator, or required dependencies should be compiled.

None of the other modules should be necessary. Models should only depend on models (and annotators), if some other dependency is required, there must be some error in the pom.xml definition of the affected extension.